### DISCUSSION TITLE
Add Bionomia integration

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5811] [Feature Requests]
Bionomia is a database and database entry tool which permits the name strings of collectors, and of taxonomists who determine specimen data, to be assigned to the unique person who collected or identified the specimen. [Wikipedia](https://en.wikipedia.org/wiki/Bionomia)

**Comment by melton-jason** [comment]
Do you have any ideas or vision relating to this integration?  
Would Bionomia be 'integrated' similarly to GBIF/Symbiota (that is, through something like Specify Network)? 
Would it be subtler, like the MorphBankView table? 
Or did you have something else in mind entirely? 

It looks Bionomia allows association between people and their accomplishments in the space of biological collections. 
Could this not already be accomplished through the Agent Identifier table? 

I would like to hear what you have in mind for this!

**Comment by dshorthouse** [comment]
Bionomia maintainer & developer here. If your goal might be to turn strings to things (Agents to obtain unique identifiers w/ ORCID or Wikidata Q number #3555, perhaps to help inform local merge decisions) then "integration" might be as simple as a user-triggered front-end to query Bionomia: https://bionomia.net/developers

**Comment by melton-jason** [comment]
Thank you for the response @dshorthouse!
That seems simple enough!
I appreciate the developer resource as well; we will definitely investigate all Bionomia has to offer when we discuss design and implementation details.
